### PR TITLE
Cut down the valid VM types

### PR DESCRIPTION
**Pull request by thekad**

### CODE REVIEW

**Comment by thekad** [comment]
@jim-minter waiting for the final list to finalize

File: pkg/api/validate/types.go [hunk 1]
```diff
@@ -88,27 +78,6 @@ var (
 		api.StandardE16sV3: {},
 		api.StandardE20sV3: {},
```
Comment (jim-minter) [inline pkg/api/validate/types.go]: not in the list - remove

File: pkg/api/validate/types.go [hunk 2]
```diff
@@ -88,27 +78,6 @@ var (
...
 		api.StandardE32sV3: {},
-		api.StandardE64sV3: {},
-
-		// Skipping StandardM* for now as they may require significant extra
-		// effort/spend to certify and support.
-
-		api.StandardGS2: {},
-		api.StandardGS3: {},
-		api.StandardGS4: {},
-		api.StandardGS5: {},
```
Comment (jim-minter) [inline pkg/api/validate/types.go]: please globally remove all the consts associated with all these removed types

**Comment by jim-minter** [comment]
@thekad please fix this up and mark it un-draft, given our timelines I want to merge this tomorrow.

**Pull request merged**